### DISCUSSION TITLE
Can I create a form to populate an Article or a Widget?

### DISCUSSION THREAD

**Discussion by srcarney** [templaza/astroid-framework#757] [Q&A]
I am wondering if there is a way that I could create a form and have it be published as an article upon approval? Or else could it be published as a widget of some sort that could be easily embedded within an article?

If so, would I be also be able to use custom fields and have them display within the Joomla Smart Search?

Thank you very much,

Sean Carney

**Comment by sonvnn** [comment]
At the moment, you can use Article Layout do to this. This is an example https://demo.astroidframe.work/article-layouts/tours-example/danube-by-bike-and-boat

**Comment by srcarney** [comment]
That looks really good. Is there any way that I could see the form that was
used for this?  I want to be sure that I will be able to create a form that
I or others can fill out to populate all of the data.

BTW, I notice that you seem to be the main person keeping Astroid Framework
alive. Do you also have a team helping with this. Is there now a open
source community involved in maintaining this?  It must be a lot of work
for you.

And, can you tell me which template houses are using Astroid Framework so I
can see some sample templates.

Thanks, Sean

PS. I did play with Astroid Framework years ago and really liked it.

On Thu, Aug 15, 2024 at 10:17 PM Sonny Le ***@***.***> wrote:

> At the moment, you can use Article Layout do to this. This is an example
> https://demo.astroidframe.work/article-layouts/tours-example/danube-by-bike-and-boat
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/templaza/astroid-framework/discussions/757#discussioncomment-10354235>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA4PRKXHGUPVEJ2EYAXM25TZRVVMHAVCNFSM6AAAAABMTI47GKVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMZVGQZDGNI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 

Faith - Family - Friends - Fun.
Take time for all four. Then back to work!

**Reply by sonvnn** [reply]
You can find some instructions about Article Layout here https://docs.astroidframe.work/article-blog/article-layout-builder

Yes, I am working on the Astroid Framework. But I am not alone. I have many people here to help support Astroid. You can check in the Discussions tab and the Issues tab to see how we are progressing. The Astroid community is still there. It is not dead. It is alive and growing very fast.. I am proud of that.

**Comment by Chacapamac** [comment]
> And, can you tell me which template houses are using Astroid Framework so I
> can see some sample templates.

https://astroidframe.work/partners

**Comment by srcarney** [comment]
Thank you very much,

Sean Carney

On Mon, Aug 19, 2024 at 8:19 AM Chacapamac ***@***.***> wrote:

> And, can you tell me which template houses are using Astroid Framework so I
> can see some sample templates.
>
> https://astroidframe.work/partners
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/templaza/astroid-framework/discussions/757#discussioncomment-10383883>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA4PRKXPLVPRYLNB63U3B3LZSHWEXAVCNFSM6AAAAABMTI47GKVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMZYGM4DQMY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 

Faith - Family - Friends - Fun.
Take time for all four. Then back to work!

**Comment by srcarney** [comment]
I am so happy to have a good framework with active development. As soon as
I can start web publishing again, I will be using Astroid. The sooner the
better.

On Fri, Aug 16, 2024 at 8:55 PM Sonny Le ***@***.***> wrote:

> You can find some instructions about Article Layout here
> https://docs.astroidframe.work/article-blog/article-layout-builder
>
> Yes, I am working on the Astroid Framework. But I am not alone. I have
> many people here to help support Astroid. You can check in the Discussions
> tab and the Issues tab to see how we are progressing. The Astroid community
> is still there. It is not dead. It is alive and growing very fast.. I am
> proud of that.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/templaza/astroid-framework/discussions/757#discussioncomment-10363896>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA4PRKQSWGZR5ARUOWOMABDZR2USBAVCNFSM6AAAAABMTI47GKVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMZWGM4DSNQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 

Faith - Family - Friends - Fun.
Take time for all four. Then back to work!

**Reply by Chacapamac** [reply]
Great, have a Good Day! 🚀
